controllers/azuresql_combined_test.go

```diff
@@ -106,7 +109,31 @@ func TestAzureSqlServerCombinedHappyPath(t *testing.T) {
 				},
 			}
 
-			EnsureInstance(ctx, t, tc, sqlDatabaseInstance)
+			EnsureInstance(ctx, t, tc, sqlDatabaseInstance1)
+		})
+
+		t.Run("set up database in primary server using sku with maxsizebytes", func(t *testing.T) {
```

Comment:
this is the secondary server surely :)

Reply:
So it is

Reply:
Ah sorry, no it's not.

It's the secondary _database_ in the primary _server_ (there are two servers too)